.github/workflows/unit_tests.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           - build: windows
             os: windows-latest
             SKIP_LIGHTGBM: False
-        python-version: [3.6, 3.7, 3.8]
```

Comment:
Lets also include it into the workflow in cronjob_unit_tests.yml file

Reply:
I've implemented your suggestion and pushed it to here!